app/kumactl/cmd/install/install_crds.go

```diff
@@ -0,0 +1,145 @@
+package install
+
+import (
+	"context"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kumahq/kuma/app/kumactl/pkg/install/k8s"
+	controlplane "github.com/kumahq/kuma/app/kumactl/pkg/install/k8s/control-plane"
+
+	k8s_apixv1beta1client "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/typed/apiextensions/v1beta1"
+
+	"github.com/kumahq/kuma/app/kumactl/pkg/install/data"
+)
+
+const KumaCrdApiGroup = "kuma.io"
+
+type InstallCrdsArgs struct {
+	NotInstalled bool
+}
+
+func newInstallCrdsCmd() *cobra.Command {
+	args := InstallCrdsArgs{}
+
+	cmd := &cobra.Command{
+		Use:   "crds",
+		Short: "Install Kuma Custom Resource Definitions on Kubernetes",
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			wantCrdFiles, err := GetCrdsFiles()
+			if err != nil {
+				return errors.Wrap(err, "Failed to read CRD files")
+			}
+
+			if !args.NotInstalled {
+				singleFile := data.JoinYAML(wantCrdFiles)
+
+				if _, err := cmd.OutOrStdout().Write(singleFile.Data); err != nil {
+					return errors.Wrap(err, "Failed to output rendered resources")
+				}
+
+				return nil
+			}
+
+			crdsToInstallMap, err := mapCrdNamesToFiles(wantCrdFiles)
+			if err != nil {
+				return errors.Wrap(err, "Failed mapping CRD files with CRD names")
+			}
+
+			kubeClientConfig, err := k8s.DefaultClientConfig()
+			if err != nil {
+				return errors.Wrap(err, "Could not detect Kubernetes configuration")
```

Comment:
Please use the same messaging in other parts of the code. We should advise the user what to do to make this work. I do realize this will be mostly used by the helm hook, still, we shall use the same messaging.

Reply:
OK I stand corrected here, I was referring to this one here: https://github.com/kumahq/kuma/blob/master/app/kumactl/cmd/install/install_control_plane.go#L115